src/test/java/org/openrewrite/java/liberty/RemoveWas2LibertyNonPortableJndiLookupTest.java

```diff
@@ -76,4 +77,240 @@ public void doX() {
         );
     }
 
+    @Test
+    void variableTest() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              package com.ibm;
+
+              import java.util.Hashtable;
+              import javax.naming.InitialContext;
+
+              public class ServerNameUsage {
+
+                  public void doX() {
+                      Hashtable<String, String> env = new Hashtable<String, String>();
+                      String initial = "java.naming.factory.initial";
+                      String url = "java.naming.provider.url";
+                      env.put(initial, "com.ibm.websphere.naming.WsnInitialContextFactory");
+                      env.put(url, "corbaloc:iiop:localhost:2809");
+                      env.put("valid", "valid");
+
+                      InitialContext ctx = new InitialContext(env);
+                  }
+
+              }
+              """,
+            """
+              package com.ibm;
+
+              import java.util.Hashtable;
+              import javax.naming.InitialContext;
+
+              public class ServerNameUsage {
+
+                  public void doX() {
+                      Hashtable<String, String> env = new Hashtable<String, String>();
+                      String initial = "java.naming.factory.initial";
```

Comment:
Perhaps we can do some additional cleanup here to remove these now unused variables. It looks like the `org.openrewrite.staticanalysis.RemoveUnusedLocalVariables` recipe could do this for us